data-raw/helper_files.R

```diff
@@ -22,6 +22,6 @@ saveRDS(dat1, "tests/testthat/helper_dat.RDS")
 
 input <- list(country = rl, language = rl2)
 createRecodeDB(input, filePath = "tests/testthat/helper_recodeDB.xlsx")
+createRecodeDB(input, filePath = "inst/extdata/example_recodeDB.xlsx")
```

Comment:
Shouldn't we create distinct files for each use case, instead of using different excel sheets?

Reply:
(Answering to both threads here)

Currently, when `eatRecode` refers to a recode data base, this refers to an `.xlsx` file with one or multiple sheets containing look up tables. This is, for instance, implemented in the `createRecodeDB()` and the `getRecodeDB()` functions (which directly read or write `.xlsx` files).

We made the decision in favor of `.xlsx` when creating the package, but discussed alternatives as well (mainly `.csv` or "proper" data base formats such as `SQLite`). Downsides to .csv from my perspective are:
- difficult to inspect and modify with Excel/without R
- issues with German vs English standards
- each look up table is by definition a single file; look up tables can not be as conveniently stored in a single file (without extending the columns), therefore the term data base seems even more misleading to me than in the `.xlsx` case

However, there are certainly advantages of .csv and other alternatives as well (as you mentioned). It would definetely be possible to rewrite/restructure the package in favor of an alternative data format and I think when agreeing on .xlsx we explicitly decided to re-evaluate this later on.

Regardless, in my opinion this is a separate discussion from this issue/PR. The current PR adds an example data base (the title of the PR may be misleading) with two look up tables to the package (in the existing framework). Discussions on alternative file formats should potentially be done together with @enkeflor and @liebelta in a separate issue.

Reply:
Yes I agree, that's another discussion. I've opened an issue.